docs/internal-documentation/maintenance/ReleasePlan.md

```diff
@@ -16,18 +17,21 @@ the respective "release schedule"
 The minor or major features that are part of each release are determined by the development team.
 
 ## Release schedule
-* a. Minor release schedule (TBD)
-* b. Major release schedule (TBD)
+The release schedule is to be created based on _minor releases_ of the Zonemaster parts, i.e. releases that changes then functionality but not the outward-facing API.
+
+When needed, _minor releases_ are changed to _major releases_ of one or more parts, i.e. when functionality changes require changes to the outward-facing API.
```

Comment:
"when functionality changes require changes to the outward-facing API" -> "when we make breaking changes"